abstractions/ruby/microsoft_kiota_abstractions/lib/microsoft_kiota_abstractions/authentication_provider.rb

```diff
@@ -0,0 +1,7 @@
+module MicrosoftKiotaAbstractions
```

Comment:
I thought we said the package name and the module name would match? isn't the package name Microsoft_kiota_abstractions?

Reply:
after internal review, we'll keep it this way for now. resolving